tests/manage/test_pvc_disruptive.py

```diff
@@ -21,43 +61,171 @@ class BaseDisruption(ManageTest):
     pvc_obj = None
     storage_type = None
 
+    def create_pvcs(self):
+        """
+        Create PVCs
+
+        Returns:
+            list: List of PVC objects
+        """
+        # Parameters for PVC yaml as dict
+        pvc_data = load_yaml_to_dict(constants.CSI_PVC_YAML)
+        pvc_data['metadata']['namespace'] = self.namespace
+        pvc_data['spec']['storageClassName'] = self.sc_obj.name
+        pvc_data['metadata']['name'] = helpers.create_unique_resource_name('test', 'pvc')
+
+        # Create 5 PVCs
+        pvc_objs = create_multiple_pvc(5, pvc_data)
```

Comment:
you can just use this function and add a new parameter sc_name

Reply:
This function is will be later replaced with new [function](https://github.com/red-hat-storage/ocs-ci/blob/90d7e6edb88b23c57a996bcd2fd383f19fa243dd/tests/helpers.py#L305-L328) once the PR 402 is merged.

Reply:
Changed to new method